general/README.md

```diff
@@ -249,6 +250,42 @@ if (isAllowed) {
 
 **[⬆ back to top](#table-of-contents)**
 
+## Functions
+
+### Immediately-invoked function expressions
+
+Avoid using IFFEs (immediately-invoked function expressions) if possible. The `bind` function is preferred when dealing with variable scope in closures (note that `bind` is only available in ES5+).
+
+```js
+// good (using .bind)
```

Comment:
we should put something more generic in the function. maybe doing something more trivial ?